.eslintrc

```diff
@@ -14,3 +14,4 @@ rules:
   no-use-before-define: [warn, { functions: false }]
   react/no-multi-comp: [off]
   no-throw-literal: [off]
+  import/no-unresolved: [warn]
```

Comment:
I'm still seeing the issue. Did a complete reinstall after rebasing and pinning the new dependencies. Then I tried upgrading ESLint. Ultimately committed with a `--no-verify` to see what Circle thinks.